docs/documentation.rst

```diff
@@ -10,13 +10,12 @@ Documentation of Plone
 The comprehensive resource for Plone documentation is http://docs.plone.org/. The following sections among 
 others are included: 
 
-* `User Manual <http://docs.plone.org/working-with-content/index.html>`_
-* `Installing Plone <http://docs.plone.org/manage/installing/index.html>`_
-* `Theme Reference <http://docs.plone.org/adapt-and-extend/theming/index.html>`_
-* `Developer Manual <http://docs.plone.org/develop/index.html>`_
+* :doc:`/working-with-content/index`
```

Comment:
please use explicit `name <location>` syntax, as sphinx does not always pick up the title right, and it can be awkward if somebody changes the title. So here: 
```
:doc:`User Manual </working-with-content/index>`
```